common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -193,6 +230,12 @@ def create_response_element(self, **kwargs):
             responseparam_element = etree.SubElement(response_element, 'responseparam')
             responseparam_element.set('type', 'tolerance')
             responseparam_element.set('default', str(tolerance))
+            if partial_range is not None and 'close' in credit_type:
+                responseparam_element.set('partial_range', str(partial_range))
+
+        if partial_answers is not None and 'list' in credit_type:
+            responseparam_element = etree.SubElement(response_element, 'responseparam')
```

Comment:
@Colin-Fredericks I suspect this pylint error is actually present elsewhere in the file; try running simply `pylint path/to/file` locally to see all the pylint errors in the whole file. It would be great if you cleaned up identical violations in otherwise untouched lines (straightening up as you go) ,but it's not required.

That said this is a weird pylint error. @benpatterson do you know anything about this?